frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -71,14 +72,26 @@ const GitSection: React.FC<GitSectionProps> = ({ showSample }) => {
         required
       />
       {values.git.showGitType && (
-        <DropdownField
-          name="git.type"
-          label="Git Type"
-          items={GitTypes}
-          title={GitTypes[values.git.type]}
-          fullWidth
-          required
-        />
+        <>
+          <DropdownField
+            name="git.type"
+            label="Git Type"
+            items={GitReadableTypes}
+            title={GitReadableTypes[values.git.type]}
+            fullWidth
+            required
+          />
+          {!gitTypeTouched && (
+            <Alert
+              isInline
+              className="co-alert"
+              variant="info"
+              title="Defaulting Git Type to Other."
```

Comment:
Please remove the period of the title.

Reply:
I just used exactly what Serena gave me. But I agree it looks odd.